baseband/helpers/sequentialfile.py

```diff
@@ -7,13 +7,27 @@
 import re
 import itertools
 from bisect import bisect
+from io import BufferedReader
 import numpy as np
 from astropy.utils import lazyproperty
 
-__all__ = ['FileNameSequencer', 'SequentialFileReader', 'SequentialFileWriter',
+__all__ = ['FileReader', 'FileNameSequencer', 'SequentialFileReader', 'SequentialFileWriter',
            'open']
 
 
+def fr_open(path, mode='rb'):
```

Comment:
Since `fr_open` and `FileReader` are used only by the ASP format, they should go in the ASP folder, not here.